ivi-sdk-java/src/main/java/games/mythical/ivi/sdk/client/model/IVIUpholdCryptoCard.java

```diff
@@ -0,0 +1,38 @@
+package games.mythical.ivi.sdk.client.model;
+
+import games.mythical.ivi.sdk.exception.IVIException;
+import games.mythical.ivi.sdk.proto.api.wallet.UpholdCryptoCard;
+import lombok.Builder;
+import lombok.Data;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@Builder
+public class IVIUpholdCryptoCard {
+    private String upholdId;
+    private String currency;
```

Comment:
got one more "label" field I just added 😅

Reply:
Do we want to go ahead and add that to the SDK now?